boot/freeldr/freeldr/bootmgr.c

```diff
@@ -26,14 +26,14 @@
 
 typedef
 VOID
-(*OS_LOADING_METHOD)(IN OperatingSystemItem* OperatingSystem,
-                     IN USHORT OperatingSystemVersion);
+(*OS_LOAD_PROC)(IN OperatingSystemItem* OperatingSystem,
```

Comment:
What's the point of this change?

Reply:
It will go together with another change not committed yet (it's WIP as the commit title says).